tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/SilentAuthTests.cs

```diff
@@ -18,6 +18,10 @@ public class SilentAuthTests
     {
         private static readonly string[] s_scopes = { "User.Read" };
 
+        /// <summary>
+        /// Initialized by MSTest (do not make private or readonly)
+        /// </summary>
+        public TestContext TestContext { get; set; }
```

Comment:
Why do you need the test context?

Reply:
i need it for the device code tst..which i've now added

---
In reply to: [348463615](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1511#discussion_r348463615) [](ancestors = 348463615)